tests/Runner/FileFilterIteratorTest.php

```diff
@@ -188,4 +188,31 @@ public function testInvalidIterator()
 
         iterator_to_array($filter);
     }
+
+    public function testFileIsAcceptedAfterFilteredAsSymlink()
```

Comment:
I think this is a unix-only feature, although I don't know much about symlinks in Windows

Reply:
:+1: 
hmm maybe I could refactor making the symlink on the fly so the tests passes on all OS'es, not sure  though...  maybe someone with more Windows knowledge can assist here

Reply:
it would be `mklink` on Windows, but I'm not convinced it's worth to pursuit

in the end, we don't want to test test `is_link` logic of internal PHP function on our side